pydantic/main.py

```diff
@@ -128,6 +128,9 @@ class BaseConfig:
     json_encoders: Dict[Type[Any], AnyCallable] = {}
     underscore_attrs_are_private: bool = False
 
+    # Whether or not inherited models as fields should be reconstructed as base model
+    copy_on_model_validation: bool = False
```

Comment:
```suggestion
    copy_on_model_validation: bool = True
```
Surely we have to stick to `True` in v1.